### ISSUE TITLE
How can we format output message for the embed chat message? 

### ISSUE THREAD

**Issue by IIAXSISII** [FlowiseAI/FlowiseChatEmbed#51]
The canvas chat bubble has nicely formatted output when the bot sends the response. 
<img width="400" alt="Screenshot 2023-11-11 at 1 17 39 PM" src="https://github.com/FlowiseAI/FlowiseChatEmbed/assets/4710838/a56822c2-7d66-427d-b610-00514f3151a9">
But when I use the pop up react module from this library, the output are displayed along with markdown syntax. 
<img width="401" alt="Screenshot 2023-11-11 at 1 23 05 PM" src="https://github.com/FlowiseAI/FlowiseChatEmbed/assets/4710838/1d1d527c-745c-4dcd-910a-2444c5a9c302">

Is there a way we can enable formatting same as its displayed in canvas when the messages are displayed to the user ?

**Comment by 0xi4o** [comment]
Yes. The popup react module doesn't support markdown formatting yet, but the canvas chat bubble does. This will be fixed in an upcoming release.

**Comment by blissiraDC** [comment]
How did you change the title bar color?

**Comment by 0xi4o** [comment]
@blissiraDC You can change the title bar background and icon color in the configuration:
```
theme: {
    button: {
	backgroundColor: '#000',
	iconColor: '#fff'
    }
}
```

**Comment by 0xi4o** [comment]
Fixed in #75
